### PR TITLE
Completed css-positioning Exercise - Bingle Padilla

### PR DESCRIPTION
**Pull request by codewithbingle**
css-positioning-relative
css-positioning-absolute gif
css-positioning-fixed

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  text-transform: uppercase;
+
+  /* your code here */
+  position: relative;
+  top: 200px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+
```
Comment (Zhiwutian) [inline css-positioning/absolute.css:39]: the overlay is not currently covering the element, make sure to set position absolute here, and check the size of the element in the inspector to make sure its the right size to cover the element

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
```
Comment (Zhiwutian) [inline css-positioning/absolute.css:6]: you need to have the proper position property assigned here

File: css-positioning/absolute.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: relative;
```
Comment (Zhiwutian) [inline css-positioning/absolute.css:33]: this should be position absolute

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates

File: css-positioning/absolute.css (hunk 3 above)
Comment (codewithbingle) [reply css-positioning/absolute.css]: Updated

File: css-positioning/absolute.css (hunk 2 above)
Comment (codewithbingle) [reply css-positioning/absolute.css:6]: Updated

File: css-positioning/absolute.css (hunk 1 above)
Comment (codewithbingle) [reply css-positioning/absolute.css:41]: Updated

Comment (Zhiwutian) [reply css-positioning/absolute.css:41]: > Updated

almost there, now you just need the css to make sure the overlay element is the right size to cover the meme

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
👍🏼

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**